### PR TITLE
mock docs update

### PR DESCRIPTION
**Pull request by hughleo**
## Proposed changes

[//]: # I've only used it on one vendor(browserstack), but mocking will work on cloud as well as locally

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/docs/MocksAndSpies.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ WebdriverIO comes with built in support for modifying network responses that all
 
 :::info
 
-This feature is currently only supported when running local tests on Chrome. It is planned to be supported on [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1591389) and [Sauce Labs](https://saucelabs.com/) soon. If you encounter problems using it please file [an issue](https://github.com/webdriverio/webdriverio/issues/new/choose) and let us know!
+Note that using the `mock` command requires support for Chrome DevTools protocol.
```
Comment (christian-bromann) [inline website/docs/MocksAndSpies.md:10]: We should add a note that cross browser support is expected once required primitives landed in WebDriver Bidi and got implemented in respective browser.

File: website/docs/MocksAndSpies.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ WebdriverIO comes with built in support for modifying network responses that all
...
+That support is given if you run tests locally in Chromium based browser or if
+you use a Selenium Grid v4 or higher. If you use a cloud vendor for spawning browsers, make sure your vendor also supports the Chrome DevTools protocol.
```
Comment (christian-bromann) [inline website/docs/MocksAndSpies.md:12]: Do we know which cloud vendor currently support DevTools in the cloud? I wonder if we can just say Sauce and Browserstack support it which would cover probably the majority of users.

**Comment by christian-bromann** [comment]
@hughleo thanks for the contribution. Mind signing the CLA?

**Comment by christian-bromann** [comment]
ping @hughleo

**Comment by rjewsbfd** [comment]
@hughleo Im currently trying to get these working with Browserstack. Did you have to do anything special to get it to work? I got information support to use the below options but I get protocol errors from puppeteer when calling commands.  Browserstack support has not been responding to my query  about the new  error. 
seleniumCdp:true,
seleniumVersion:'4.1.0',

[chrome 112.0.5615.50 WINDOWS #0-0] Protocol error (Browser.getVersion): Target closed
[chrome 112.0.5615.50 WINDOWS #0-0] TargetCloseError: Protocol error (Browser.getVersion): Target closed

**Comment by hughleo** [comment]
@rjewsbfd  I saw something similar, so just updated the seleniumVersion (  '4.15.0',)

File: website/docs/MocksAndSpies.md (hunk 2 above)
Comment (hughleo) [reply website/docs/MocksAndSpies.md:12]: lambdatest too from what I understand

**Comment by christian-bromann** [comment]
@hughleo any chance you can sign the CLA? I can't merge the PR otherwise.

**Comment by rjewsbfd** [comment]
> @rjewsbfd I saw something similar, so just updated the seleniumVersion ( '4.15.0',)

@hughleo thanks.!!! I didn't go high enough when i was playing with the version number but after updating wdio and the selenium version it worked.

**Comment by hughleo** [comment]
> @hughleo any chance you can sign the CLA? I can't merge the PR otherwise.

trying to....

**Comment by christian-bromann** [comment]
> trying to....

@hughleo anything I can do to help? Note that the email you use to sign the CLA needs to be the same email you use in your git settings.

**Comment by christian-bromann** [comment]
ping @hughleo

**Comment by christian-bromann** [comment]
I went ahead and adapted the changes to `main`. Please make sure to use the email you use to sign the CLA for all your commits next time so we don't run into any issues with the bot. Thank you for the contribution!

**Pull request closed without merging**